### ISSUE TITLE
[Issue]: Identify search empty result 

### ISSUE THREAD

**Issue by Askejm** [jellyfin/jellyfin#9854]
### Please describe your bug

Whenever I add a movie or show, the cover art and that sort of metadata doesn't appear. It used to some of the time, but now it never does. However, when I use the identify tool I get no results 
![image](https://github.com/jellyfin/jellyfin/assets/55248977/bc8fe12f-c885-4e7f-bd1d-0e97146aca66)
I tried using every identification method, name, year, imdb id etc, no results. I did enable metadata downloaders for this library 
![chrome_OTjaOBKpIt](https://github.com/jellyfin/jellyfin/assets/55248977/bf07cdbe-ed6a-465a-86f3-d756a04b5406)

### Jellyfin Version

10.8.z

### Environment

```markdown
- OS: Windows
- Linux Kernel: None
- Virtualization: None
- Clients: Browser, jellyfin and swiftfin app
- Browser: Chrome and firefox
- FFmpeg Version: 
- Playback Method:
- Hardware Acceleration: Nvenc
- GPU Model: 3090 
- Plugins: None
- Reverse Proxy: Nginx proxy manager
- Base URL: 
- Networking: 
- Storage: Local
```

### Code of Conduct

- [x] I agree to follow this project's Code of Conduct

**Comment by Askejm** [comment]
It appears to be an issue with my VPN. Currently, it only works when my VPN (NordVPN) is turned completely off. I tried using split tunneling on all exe files in the Jellyfin server folder, but it still doesn't work, only when my VPN is completely off. If anyone knows how to fix this without turning my VPN off completely, or what to split tunnel i would appreciate that

**Comment by Bond-009** [comment]
Not a Jellyfin issue.

**Comment by Askejm** [comment]
????

**Comment by Bond-009** [comment]
> It appears to be an issue with my VPN. Currently, it only works when my VPN (NordVPN) is turned completely off. I tried using split tunneling on all exe files in the Jellyfin server folder, but it still doesn't work, only when my VPN is completely off. If anyone knows how to fix this without turning my VPN off completely, or what to split tunnel i would appreciate that

You said so yourself, am I missing something?

**Comment by zQueal** [comment]
Also having this issue. Not connected to a VPN. Wired directly to the internet.

🤷🏻‍♂️

**Comment by bmcgonag** [comment]
I'm experiencing this as well. No VPN. Directly connected via ethernet.

**Comment by zQueal** [comment]
I ended up disabling Metadata from within Jellyfish and using TinyMediaManager to edit my library via "metadata savers" being set to "nfo". Occationally, even though metadata fetching is disabled Jellyfin still messes with the saved metadata from TinyMediaManager, but it's a way better setup. I highly suggest it.

**Comment by superjax** [comment]
I'm also seeing this directly over ethernet.

(server is running Ubuntu 22.04, jellyfin-server is at `10.9.9.0`)

**Comment by brunofin** [comment]
Same problem for me, no VPN, connected directly to the internet. It is running on podman and on a custom network. Jellyseerr is in that same network and it works fine as far as I can tell.

I will try to get some logs later.

**Comment by brunofin** [comment]
I get this every time I try to identify:

```
[18:29:44] [INF] [27] Emby.Server.Implementations.HttpServer.WebSocketManager: WS 34.198.71.98 request

[18:29:59] [ERR] [59] MediaBrowser.Providers.Manager.ProviderManager: Provider TheMovieDb failed to retrieve search results

System.Net.Http.HttpRequestException: Resource temporarily unavailable (api.themoviedb.org:443)

 ---> System.Net.Sockets.SocketException (11): Resource temporarily unavailable

   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)

   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)

   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|285_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)

   --- End of inner exception stack trace ---

   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(QueueItem queueItem)

   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)

   at System.Net.Http.DiagnosticsHandler.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)

   at TMDbLib.Rest.RestRequest.SendInternal(HttpMethod method, CancellationToken cancellationToken)

   at TMDbLib.Rest.RestRequest.Get[T](CancellationToken cancellationToken)

   at TMDbLib.Rest.RestRequestExtensions.GetOfT[T](RestRequest request, CancellationToken cancellationToken)

   at TMDbLib.Client.TMDbClient.GetConfigAsync()

   at MediaBrowser.Providers.Plugins.Tmdb.TmdbClientManager.EnsureClientConfigAsync()

   at MediaBrowser.Providers.Plugins.Tmdb.TmdbClientManager.SearchMovieAsync(String name, Int32 year, String language, CancellationToken cancellationToken)

   at MediaBrowser.Providers.Plugins.Tmdb.Movies.TmdbMovieProvider.GetSearchResults(MovieInfo searchInfo, CancellationToken cancellationToken)

   at MediaBrowser.Providers.Manager.ProviderManager.GetRemoteSearchResults[TItemType,TLookupType](RemoteSearchQuery`1 searchInfo, BaseItem referenceItem, CancellationToken cancellationToken)

[18:29:59] [ERR] [42] MediaBrowser.Providers.Manager.ProviderManager: Provider The Open Movie Database failed to retrieve search results

System.Net.Http.HttpRequestException: Resource temporarily unavailable (www.omdbapi.com:443)

 ---> System.Net.Sockets.SocketException (11): Resource temporarily unavailable

   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)

   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)

   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|285_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)

   at Jellyfin.Networking.HappyEyeballs.HttpClientExtension.AttemptConnection(AddressFamily addressFamily, SocketsHttpConnectionContext context, CancellationToken cancellationToken)

   at Jellyfin.Networking.HappyEyeballs.HttpClientExtension.OnConnect(SocketsHttpConnectionContext context, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)

   --- End of inner exception stack trace ---

   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(QueueItem queueItem)

   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)

   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)

   at System.Net.Http.DiagnosticsHandler.SendAsyncCore(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at System.Net.Http.DecompressionHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)

   at Microsoft.Extensions.Http.Logging.LoggingHttpMessageHandler.<SendCoreAsync>g__Core|5_0(HttpRequestMessage request, Boolean useAsync, CancellationToken cancellationToken)

   at Microsoft.Extensions.Http.Logging.LoggingScopeHttpMessageHandler.<SendCoreAsync>g__Core|5_0(HttpRequestMessage request, Boolean useAsync, CancellationToken cancellationToken)

   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)

   at MediaBrowser.Providers.Plugins.Omdb.OmdbItemProvider.GetSearchResultsInternal(ItemLookupInfo searchInfo, Boolean isSearch, CancellationToken cancellationToken)

   at MediaBrowser.Providers.Manager.ProviderManager.GetRemoteSearchResults[TItemType,TLookupType](RemoteSearchQuery`1 searchInfo, BaseItem referenceItem, CancellationToken cancellationToken)
   ```

**Comment by zQueal** [comment]
> Resource temporarily unavailable (api.themoviedb.org:443)

API issue.

**Comment by brunofin** [comment]
> > Resource temporarily unavailable (api.themoviedb.org:443)
> 
> API issue.

Yea I saw that, but is this something I need to solve from my end or is it a jellyfin bug? I haven't been able to identify in weeks.

**Comment by zQueal** [comment]
> System.Net.Http.HttpRequestException: Resource temporarily unavailable (api.themoviedb.org:443)

It's not a jellyfin bug. There's either an issue on the themoviedb.org side (maybe they IP banned you or something?) or perhaps your ISP has blocked connections to themoviedb.org. Do you have a network wide adblockers? Private DNS?

Can you connect to themoviedb.org in a browser?

Try using curl to send a direct request via `api.themoviedb.org`. Instead of using the port access `:443`, try just using `https://api.themoviedb.org/`. They may be serving HTTPS on `:80` and not `:443`. If that's the case your query will fail. It _may_ even present as a network issue like you're seeing.

**Comment by brunofin** [comment]
So I figured out mt issue, and yes it was very specific to my network setup. Both my servarr stack and my nextcloud podman networks suffered from the same issue, which is that in both cases, I assigned a container's ip address to the network's gateway address (`x.x.x.1`). This obviously would cause all containers to lose internet access.

But that's also a cool lesson, because I learned that if I WANT, I create a container to be the network's gateway and do custom stuff with that which will affect all containers in that network. Neat..

Anyway, thanks for jumping in and trying to help anyway, that's appreciated.
